### ISSUE TITLE
DEFAULT_TENANT is not created in local deployment

### ISSUE THREAD

**Issue by fkaltner** [eclipse/hono#1599]
**What I did:**
I followed the steps for the Helm based deployment (see: https://www.eclipse.org/hono/docs/deployment/helm-based-deployment/) on macOS using a Kubernetes cluster running in Docker Desktop.

**What I observed:**
The curl command for checking the existence of the DEFAULT_TENANT returns a 404.

**What I think what went wrong:**
The device registry could not process the payload example-tenants.json contained in the Eclipse Hono 1.0.0 Archive (see: https://www.eclipse.org/hono/downloads/)

**What I think may be a solution for the problem:**
With the implementation of the Support for multiple GAs in Registry and Tenant APIs (I think this is #623) the type of field `trustedCertificateAuthorities` in the class Tenant was changed from `TrustedCertificateAuthorities` to `List<TrustedCertificateAuthorities>`, see revision 3f4b120. The Unit Test runs fine, since its fixture in tenants.json does not contain a trusted CA, i.e. the change made was not covered by the unit test.

As far as I can see adapting the JSON in the unit test to contain a trusted CA and in the Eclipse Hono Archive to the changes made in 3f4b120 should fix the problem.

**Comment by kaniyan** [comment]
@fkaltner Thats a good catch :+1: and thanks for reporting.

**Comment by sophokles73** [comment]
@fkaltner can you check if this works using master?

**Comment by fkaltner** [comment]
I just built locally on my machine with `mvn clean install -Pbuild-docker-image,metrics-prometheus`, cd'ed to `hono/deploy/target` and ran `tar xvzf eclipse-hono-1.1.0-SNAPSHOT-chart.tar.gz`.

Then cd'ed to `eclipse-hono-1.1.0-SNAPSHOT` and ran `helm install --dep-up --name hono --namespace hono eclipse-hono/`

Everything came up and I was able to successfully query the DEFAULT_TENANT.

Thanks for fixing! 👍
